www/index.html

```diff
@@ -977,7 +977,22 @@
           {{/each image}}
         </div>
       </script>
-  </section>
+
+      <script type="text/template7" id="checkTermsVersionTemplate">
+        <div class="popup popup-terms">
+          <div class="background-overlay"></div>
+           <div class="block">
```

Comment:
Think it would be nice to add a title above the block div saying "Sekretessvillkor" or something.